float32 right_wheel_velocity
float32 right_wheel_position
float32 left_wheel_velocity
float32 left_wheel_position
